modules/common/src/main/resources/org/opencastproject/util/MimeTypes.xml

```diff
@@ -19,6 +19,11 @@
     <Description>JavaScript Object Notation</Description>
     <Extensions>json</Extensions>
   </MimeType>
+  <MimeType>
+    <Type>application/octet</Type>
+    <Description>MH 1.3 CaptureAgent MPEG media</Description>
+    <Extensions>mpeg,mpg,m1s,m75,m15,mpm,mpv</Extensions>
```

Comment:
Shouldn't `*.mpg` or `*.mpeg` have a mime type like `video/mpeg` or `video/mpg`? Not sure about the rest…

Reply:
Agreed.  I don't think I've even see the other formats, aside from the occasional mpv.  @ziegenberg, is this something you're running into regularly?

Reply:
`mpv` is my default media player ;-D

Reply:
I'll recheck. This fix was some suggestion from a third party.

Reply:
Reading through the Jira ticket, using `<Type>video/mpeg</Type>` should work